.vscode/extensions.json

```diff
@@ -1,3 +1,8 @@
 {
-  "recommendations": ["ms-azuretools.azure-dev", "angular.ng-template", "42Crunch.vscode-openapi"]
-}
+  "recommendations": [
+    "ms-azuretools.vscode-azurefunctions",
```

Comment:
This extension is generating a lot of additional configuration. Can you please remove it from the recommendations and revert the `tasks.json` file? Thanks!

Reply:
Okay, I've removed the extension, but about reverting, I haven't dived deeply into it and wouldn't want to add issues to the existing codebase. Could you please provide hints or a guide that I can follow to successfully revert the `tasks.json` file? @anfibiacreativa

Reply:
Found a way to do it, please check it now @anfibiacreativa

Reply:
Also reverted the `settings.json` file